src/components/main/Liquidations/Table/CustomAccountCell.tsx

```diff
@@ -0,0 +1,16 @@
+import { ExternalLink } from 'components/common/Icons'
+import Text from 'components/common/Text'
+
+interface Props {
+  value: string
+}
+
+export default function CustomAccountCell(props: Props) {
```

Comment:
I'm not happy with the prefix `Custom` if you'd be strict you would name every component in the app that doesn't come from a package `CustomComponent`.


I'd suggest to move all Cells to a `Table/cells` folder and remove the `Custom` prefix and `Cell` suffix
```suggestion
export default function Account(props: Props) {
```

Reply:
yep, that sounds better